frontend/packages/console-shared/src/components/details-page/ExtensionDetailsItem.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { K8sResourceCommon } from '@console/dynamic-plugin-sdk/src/extensions/console-types';
+import { DetailsItem as DetailsItemExtension } from '@console/dynamic-plugin-sdk/src/extensions/details-item';
+import { ResolvedExtension } from '@console/dynamic-plugin-sdk/src/types';
+import { DetailsItem } from '@console/internal/components/utils/details-item';
+
+export const ExtensionDetailsItem: ExtensionDetailsItemComponent = ({ extension, obj }) => {
+  const { path, title, component: Component, id } = extension.properties;
+  const sortWeight = extension.properties.sortWeight ?? 'none';
+  const value = _.get(obj, path);
+  if (!Component && typeof value === 'object') {
+    // eslint-disable-next-line no-console
+    console.warn(
+      `Invalid 'console.resource/details-item' extension: '${id}'. The value referenced by path '${path}' is not primitive, and no component property is provided.`,
+    );
+  }
+  const content = Component ? (
+    <Component key={extension.properties.id} obj={obj} path={path} value={value} />
```

Comment:
The `key` here is not wrong, but my first impression was: Oh it should be moved to the DetailsItem below..

But then I checked `frontend/public/components/default-resource.tsx` and saw that the loop adds already a key correctly to the usage of `<ExtensionDetailsItem`. So I think it's not needed here, and not on the DetailsItem below, or?

```suggestion
    <Component obj={obj} path={path} value={value} />
```

Reply:
Oh, yes, thanks for catching this. It was an oversight!